types/errors.go

```diff
@@ -0,0 +1,56 @@
+package types
```

Comment:
where did these come from?

Reply:
Came up with some that might be useful to have, there are none defined anywhere else